### DISCUSSION TITLE
blog/v1-34-released/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#425] [Page comments]
# blog/v1-34-released/

Efficient number incrementation, a throwback to Scratch 2.0, and more!

https://scratchaddons.com/blog/v1-34-released/

**Comment by dshultz2** [comment]
Is it possible to use Scratch Addons with the Gandi IDE?

**Reply by BroJac5246** [reply]
It is not. It is difficult supporting multiple similar-but-different websites at once, so Scratch Addons only supports Scratch and also provides the addons that TurboWarp uses (though you don't have to have to extension installed to use those).
